soc/egis/soc.yml

```diff
@@ -0,0 +1,6 @@
+family:
+- name: andes_v5
```

Comment:
can't have the same family for a complete different device as an existing family